db/migrate/20181011164429_add_container_nodes.rb

```diff
@@ -0,0 +1,18 @@
+class AddContainerNodes < ActiveRecord::Migration[5.1]
+  def change
+    create_table "container_nodes", :id => :bigserial do |t|
+      t.references "source", :type => :bigint, :index => true
+      t.string     "source_ref"
+      t.string     "resource_version"
+      t.string     "name"
+      t.integer    "cpus"
+      t.bigint     "memory"
```

Comment:
I wasn't sure if we wanted to go full-blown ComputerSystem + Hardware with this or just keep it simple, thoughts?

Reply:
yeah, lets keep it simple